xml/dns_versionbind.xml

```diff
@@ -516,6 +516,7 @@
     <param pos="0" name="service.family" value="NSD"/>
     <param pos="0" name="service.product" value="dnsd"/>
     <param pos="1" name="service.version"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:nlnetlabs:name_server_daemon:{service.version}"/>
```

Comment:
This change alone, via the cpe-remap change, added CPEs for over a million matches from our recent DNS study and so I wanted to get it in now.